fink_filters/filter_anomaly_notification/filter_utils.py

```diff
@@ -426,16 +426,22 @@ def get_cutout(ztf_id):
             cutout image in png format
 
     '''
+    # transfer cutout data
     r = requests.post(
-        'https://fink-portal.org/api/v1/cutouts',
-        json={
-            'objectId': ztf_id,
-            'kind': 'Science'
-        },
-        timeout=25
+        "https://fink-portal.org/api/v1/objects",
+        json={"objectId": ztf_id, "withcutouts": "True"},
```

Comment:
To avoid downloading unnecessary data, you can add the argument `"cols": "b:cutoutScience_stampData"`

Reply:
Done!